dashboard.py

```
@@ -9479,11 +9481,15 @@ def decode_filters_from_url():
-                        if tp.strip().isdigit() and int(tp.strip()) in all_tp_sizes
-                    ]
+                    parsed = []
+                    for tp in query_params["tp_sizes"].split(","):
+                        val = float(tp.strip())
+                        int_val = int(val)
+                        if int_val in all_tp_sizes:
+                            parsed.append(int_val)
+                        elif val in all_tp_sizes:
+                            parsed.append(val)
+                    url_tp_sizes = parsed
                 except:
```

Ruff (0.15.13)

[error] Do not use bare `except`

(E722)